storagemarket/deal_publisher.go

```diff
@@ -117,7 +117,7 @@ func NewDealPublisher(publishMsgCfg PublishMsgConfig) func(lc fx.Lifecycle, full
 	}
 }
 
-func newDealPublisher(
+func NewDealPublisherSimple(
```

Comment:
Why are we exporting this? I think it should stay unexported.

Reply:
Hey it's for the tests so we don't have to deal with the lifecycle stuff when writing tests for the Boost SM.

Reply:
`NewDealPublisherSimple` is used only in the same package. If we don't need a struct or func exported, it really shouldn't be exported.